app/i18n/locales/ar.js

```diff
@@ -129,8 +129,8 @@ export default {
 	creating_invite: 'إنشاء دعوة',
 	Channel_Name: 'اسم القناة',
 	Channels: 'قنوات',
-	Chats: 'محادثات',
-	Call_already_ended: 'انتهت المكالمة بالفعل!',
+	Chats: 'الرسائل',
+	Call_already_ended: 'انتهت المكالمة!',
```

Comment:
I'm reviewing with Google Translate, so it's not accurate, but it looks like the current translation is better.
The intention here is to tell the user "The call has already ended", which is precisely what the current string says.

Reply:
sometimes google give wrong translation in arabic it well the call has already ended in first translation was the call is really ended but in the string is the call already ended so i corrected it to the same meaning